src/Microsoft.TestPlatform.ObjectModel/Host/ITestRunTimeProvider.cs

```diff
@@ -65,8 +65,9 @@ public interface ITestRuntimeProvider
         /// Launches the test host for discovery/execution.
         /// </summary>
         /// <param name="testHostStartInfo">Start parameters for the test host.</param>
+        /// <param name="cancellationToken"></param>
         /// <returns>ProcessId of launched Process. 0 means not launched.</returns>
-        Task<int> LaunchTestHostAsync(TestProcessStartInfo testHostStartInfo);
+        Task<int> LaunchTestHostAsync(TestProcessStartInfo testHostStartInfo, CancellationToken cancellationToken);
```

Comment:
I think the return code is used to terminate the host later.

Will the return type be always an `int`? Could there be a scenario where the TestRuntimeProvider requires more information (like the Emulator prodess id and the testhost process id)?